### PR TITLE
feat(styles): Update buttons colors

### PR DESCRIPTION
**Pull request by imagoiq**

### CODE REVIEW

File: packages/styles/src/mixins/_color.scss [hunk 1]
```diff
@@ -24,12 +24,12 @@
 // Selector for styles to be applied on dark backgrounds
 @mixin on-dark-background($contains: false) {
   @if $contains == true {
-    &:is(#{color.$dark-backgrounds}),
-    :is(#{color.$dark-backgrounds}) & {
+    &:where(#{color.$dark-backgrounds}),
+    :where(#{color.$dark-backgrounds}) & {
```
Comment (imagoiq) [inline packages/styles/src/mixins/_color.scss:28]: Big change, this was necessary to be able to keep style for button with specific color background (e.g. `.btn-petrol-bright`). This mixin was changed recently to support `post-aler[type"..."]` and this change the specificity as it adds one element to the list of classes, and then override the style for the specific color background.

**Comment by imagoiq** [comment]
@rouvenpost I need your design review to check if it's correct and if I didn't miss any change.

**Comment by rouvenpost** [comment]
I think we shoul remove the «Accent colors» Section altogether. They should not be used anymore.

**Comment by imagoiq** [comment]
> I think we should remove the «Accent colors» Section altogether. They should not be used anymore.

You're right. They will be deprecated with this ticket: https://github.com/swisspost/design-system/issues/2863

File: packages/styles/src/components/button.scss [hunk 2]
```diff
@@ -41,71 +38,116 @@
...
   text-decoration: none !important; // For <a> tags, !important for hover
   white-space: nowrap; // Long content should never break in buttons
-}
 
-// Primary
-.btn-primary {
-  border-color: transparent;
-  background-color: var(--post-gray-80);
-  color: var(--post-contrast-color-inverted);
+  &:disabled {
```
Comment (alizedebray) [inline packages/styles/src/components/button.scss:42]: It would be good to use the `utilities.disabled-style` mixin here.

**Review by rouvenpost — APPROVED** [review]

**Pull request merged**